### PR TITLE
Enhancement Shaman APL update and Tempest buff adjustment

### PR DESCRIPTION
**Pull request by IIeTpoc**
Enhancement Shaman APL update and Tempest buff adjustment (tempest has max stacks = 10)

### CODE REVIEW

**Comment by IIeTpoc** [comment]
Tested and working. All APL changes were made with translation into addon's language

File: TheWarWithin/ShamanEnhancement.lua [hunk 1]
```diff
@@ -2028,7 +2037,7 @@ spec:RegisterAbilities( {
         end,
 
         bind = "lightning_bolt",
-        copy = 454009
+        copy = 454009, 454015
```
Comment (Hekili) [inline TheWarWithin/ShamanEnhancement.lua:2040]: Needs to be bracketed:  `{ 454009, 454015 }`

File: TheWarWithin/ShamanEnhancement.lua [hunk 2]
```diff
@@ -693,6 +693,14 @@ spec:RegisterAuras( {
         duration = 15.0,
         max_stack = 1,
     },
+    -- Talent: Tempest
+    -- https://www.wowhead.com/spell=454015/tempest
+    tempest = {
+        id = 454015,
+        duration = 30.0,
+        max_stack = 10,
+        copy = 454009, 452201
```
Comment (Hekili) [inline TheWarWithin/ShamanEnhancement.lua:702]: Needs to be bracketed:  `{ 454009, 452201 }`

File: TheWarWithin/Priorities/ShamanEnhancement.simc
Comment (Hekili) [inline TheWarWithin/Priorities/ShamanEnhancement.simc:1]: Your APL changes need to be rechecked against what I uploaded today; if you were just merging SimC updates, then these should be resolved and no longer needed.

**Comment by IIeTpoc** [comment]
I did not manage to resolve all conflicts in this PR, sorry. I made a new one with adjustments that you wrote and also checked the APL and your version after update still differs from the version in simcraft.

**Pull request closed without merging**